api/Worker.json

```
@@ -126,6 +135,59 @@
             "deprecated": false
           }
         },
+        "ecmascript_modules": {
+          "__compat": {
+            "description": "Support for ECMAScript modules",
+            "support": {
+              "chrome": {
+                "version_added": "80"
+              },
+              "chrome_android": {
+                "version_added": "80"
+              },
+              "edge": {
+                "version_added": "80"
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "12.17.0",
+                "partial_implementation": true,
+                "notes": "ECMAScript modules are enabled based on the file extension <code>.mjs</code> rather than the <code>type</code> option."
```

```suggestion
                "notes": "ECMAScript modules are enabled for files ending with <code>.mjs</code> and for files ending with <code>.js</code> when the nearest parent <code>package.json</code> file contains a top-level field <code>"type"</code> with a value of <code>"module"</code>."
```

I've borrowed part of the phrasing here [from the Node.js docs](https://nodejs.org/api/esm.html#esm_package_json_type_field).